src/Domain/ElementId.hpp

```diff
@@ -67,6 +74,10 @@ template <size_t VolumeDim>
 bool operator!=(const ElementId<VolumeDim>& lhs,
                 const ElementId<VolumeDim>& rhs) noexcept;
 
+template <size_t VolumeDim>
```

Comment:
This function already exists in Domain/InitalElementIds.hpp , although it does it for all blocks and not a specific one.

Reply:
So what is your request? Move this function to there and have the one for all blocks call this one?

Reply:
Is there a reason to call it on only one block?  If no, my suggestion is to remove the function; otherwise, yes move the function (though I would have this one call the other one)

Reply:
I use it on a single Block in some of the other evolution things, at least right now. I've added an overload to InitialElementIds and the ones for all blocks call the ones for single blocks. I've also separated this change into a separate commit.